### ISSUE TITLE
Icon-request: fa-3dprinting

### ISSUE THREAD

**Issue by Draz3l** [FortAwesome/Font-Awesome#7625]
Hello there!

I would be glad if you can add a 3d printed icon like this one:

![drawing 2](https://cloud.githubusercontent.com/assets/15016726/10341690/07638dde-6d16-11e5-8d04-52f3208ca4e8.png)

**Comment by Draz3l** [comment]
Hi again! 
How can I use this icon in "font awesome" plugin for wp ?

**Comment by tagliala** [comment]
This icon is not part of FontAwesome, you could consider a custom build: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome

**Comment by arnaudcosson** [comment]
I would love to have a 3D Printer icon too.
Hope it will be added soon.
Thanks for the amazing job you do with these icons

**Comment by daemionfox** [comment]
Another vote for this!

**Comment by JoelEB** [comment]
+1

**Comment by redoper1** [comment]
It would be really nice to have a 3D printer icon in FA.

But maybe some another icon than this initially sent, maybe something little more corresponding to current printer icon style.

**Comment by tozerpm** [comment]
Yeap, I would go even further for some tech grouping

**Comment by AlbertMN** [comment]
+1
Would be awesome!

**Comment by wengeol2** [comment]
+1 
I was surprised to see that an icon for 3D Printer isn't already here

**Comment by netpenthe** [comment]
+1 i need this

**Comment by thiagosfsilva** [comment]
+1

**Comment by MingyaoLiu** [comment]
+ 1

**Comment by LeeOtts** [comment]
Yes this needs to be a thing... having to use the scatterpoint until then

**Comment by netpenthe** [comment]
+1 please

**Comment by sm3dev** [comment]
> +1
> I was surprised to see that an icon for 3D Printer isn't already here.. something like this: https://www.vectorstock.com/royalty-free-vector/3d-print-printer-icon-isolated-contour-vector-29957826

🙌

**Comment by fuzzygroup** [comment]
As a pro customer, I'm going to cast my hat in the ring for this.  Hear Hear!

**Comment by roooodcastro** [comment]
It's 2024 and no 3D printer icon! +1 please

**Comment by netpenthe** [comment]
yes pleaes.. 3d printer icon .. camoan!

**Comment by citizen42o** [comment]
Please the 3D printer icon is needed... since 2016 https://github.com/FortAwesome/Font-Awesome/issues/9949

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by lt-kraken** [comment]
> This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.
> 
> In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.
> 
> We’ll update this issue when the icon is released into the wild 🙂

Would this only entail the 3d printer icon, or also additional icons such as filament roll, printer configuration, nozzle, perhaps filament types, bed heating, etc?

**Comment by CodeHedge** [comment]
DO IT
